gnome-image-installer/pages/install/gis-install-page.c

```diff
@@ -637,7 +635,7 @@ gis_install_page_verify (GisPage *page)
       goto out;
     }
 
-  if (!g_spawn_async_with_pipes (NULL, args, NULL,
+  if (!g_spawn_async_with_pipes (NULL, (gchar **) args, NULL,
```

Comment:
IIRC this cast is the best way to tackle this problem, from the times I’ve tried to come up with a better solution in the past.